course-material/intro-to-sql/aggregate-functions-in-sql.md

```diff
@@ -50,6 +49,7 @@ from zipfile import ZipFile
 import pandas as pd
```

Comment:
Please study how the code in the main branch is setup.

I was able to fix this so that it is encoded as an ipython cell.

Reply:
I previous versions, it was encoded under a "```python```" block, but this means it won't be executed.